src/lib/utils/getUtilityStyles.ts

```diff
@@ -1,20 +1,19 @@
 import { ViewStyle, TextStyle } from 'react-native'
 import { ITheme } from '../..'
 import { get } from 'lodash'
-import { elevationMap } from './maps/view'
 
-export const getViewUtiityStyles = (props: any) => {
+export const getViewUtiityStyles = (props: any, theme: ITheme) => {
   let styles: ViewStyle = {}
 
   if (props.center) {
     styles.justifyContent = 'center'
     styles.alignItems = 'center'
   }
 
-  if (elevationMap[props.elevation]) {
+  if (props.elevation && theme.elevation[props.elevation]) {
```

Comment:
If elevation is not defined in theme and elevation props is set to a view the app crashes. Could you add some additional check to prevent  'Cannot read property of undefined'?